ios/FluentUI/Core/ColorProviding.swift

```diff
@@ -30,9 +30,9 @@ public protocol ColorProviding {
     @objc var brandStroke1: UIColor { get }
     @objc var brandStroke1Pressed: UIColor { get }
     @objc var brandStroke1Selected: UIColor { get }
-    @objc var brandGradient1: UIColor { get }
-    @objc var brandGradient2: UIColor { get }
-    @objc var brandGradient3: UIColor { get }
+    @objc var brandGradient1: UIColor? { get }
+    @objc var brandGradient2: UIColor? { get }
+    @objc var brandGradient3: UIColor? { get }
```

Comment:
Why the change to optional? I thought we had a reason for not making the other `ColorProviding` colors optional.

Reply:
Spoke with @harrieshin offline and we thought it'd be best to minimize breaking changes when updating the apps providers.